command/command.go

```diff
@@ -135,118 +157,70 @@ func PrintableCommandArgs(isQuoteFirst bool, fullCommandArgs []string) string {
 	return strings.Join(cmdArgsDecorated, " ")
 }
 
-// RunCmdAndReturnExitCode ...
-func RunCmdAndReturnExitCode(cmd *exec.Cmd) (exitCode int, err error) {
+// Deprecated: Use Command instead.
+func runCmdAndReturnExitCode(cmd *exec.Cmd) (exitCode int, err error) {
 	err = cmd.Run()
 	exitCode = cmd.ProcessState.ExitCode()
 	return
 }
 
-// RunCmdAndReturnTrimmedOutput ...
-func RunCmdAndReturnTrimmedOutput(cmd *exec.Cmd) (string, error) {
+// Deprecated: Use Command instead.
+func runCmdAndReturnTrimmedOutput(cmd *exec.Cmd) (string, error) {
 	outBytes, err := cmd.Output()
 	outStr := string(outBytes)
 	return strings.TrimSpace(outStr), err
 }
 
-// RunCmdAndReturnTrimmedCombinedOutput ...
-func RunCmdAndReturnTrimmedCombinedOutput(cmd *exec.Cmd) (string, error) {
+// Deprecated: Use Command instead.
+func runCmdAndReturnTrimmedCombinedOutput(cmd *exec.Cmd) (string, error) {
 	outBytes, err := cmd.CombinedOutput()
 	outStr := string(outBytes)
 	return strings.TrimSpace(outStr), err
 }
 
-// RunCommandWithReaderAndWriters ...
-func RunCommandWithReaderAndWriters(inReader io.Reader, outWriter, errWriter io.Writer, name string, args ...string) error {
```

Comment:
Removed because it was [unused](https://github.com/search?q=org%3Abitrise-io+org%3Abitrise-steplib+%22RunCommandWithReaderAndWriters%22&type=Code) in external repositories.